lldb/unittests/Symbol/TestSwiftASTContext.cpp

```diff
@@ -268,19 +268,82 @@ TEST_F(TestSwiftASTContext, ApplyWorkingDir) {
   EXPECT_EQ(
       single_arg_rel_path,
       llvm::SmallString<128>("-fmodule-map-file=rel/dir/module.modulemap"));
+
+  // fmodule-file needs to handle different cases:
+  //  -fmodule-file=path/to/pcm
+  //  -fmodule-file=name=path/to/pcm
```

Comment:
here it says `-fmodule-file=name=path/to/pcm` but all the tests use `-fmodule-file=modulename=path/to/pcm` is that a tricky test for the edge-case of passing in something that ends with `name=` or a typo?

Reply:
Thanks!

Reply:
The `name` string can be anything, it is used to indicate the name of the module that you are providing the path to. I can make these the same if you think it is clearer? It shouldn't make any difference to the flag splitting logic though.

Reply:
Oh, this isn't a key=value pair whre the key is "name", it's [name of the module]=[file name]!

Reply:
Exactly, yes. I believe the idea is that the compiler can avoid loading pcm files until they are imported.